test/e2e/taints_test.go

```diff
@@ -305,7 +310,7 @@ var _ = framework.KubeDescribe("NoExecuteTaintManager [Serial]", func() {
 		framework.RemoveTaintOffNode(cs, nodeName, testTaint)
 		taintRemoved = true
 		By("Waiting some time to make sure that toleration time passed.")
-		timeoutChannel = time.NewTimer(10 * time.Second).C
+		timeoutChannel = time.NewTimer(time.Duration(2*KubeletPodDeletionDelaySeconds+AdditionalWaitPerDeleteSeconds) * time.Second).C
```

Comment:
That would mean that this test by design takes 2minutes+ - do we really need to wait that long?

What is toleration time? Can't we just wait "pod deletion delay + additional wait + toleration time"?